pkg/controller/shared_state.go

```diff
@@ -27,6 +27,7 @@ import (
 	"sync"
 	"sync/atomic"
 
+	er "github.com/pkg/errors"
```

Comment:
Just use the std errors package. This one has been archived.

Reply:
Done.